docs_src/design/adr/devops/0010-Release-Artifacts.md

```diff
@@ -0,0 +1,95 @@
+# Release Artifacts
+
+## Status
+
+In Review 06/10/2020
+
+## Context
+
+During the Geneva release of EdgeX Foundry the DevOps WG transformed the CI/CD process with new Jenkins pipeline functionality. After this new functionality was added we also started adding release automation. This new automation is outlined in ADR 0007 Release Automation. However, in ADR 0007 only three release artifact types are outlined. This document is meant to be a living document to try to outline the artifact types for an EdgeX Foundry release and should be modified as that changes.
+
+## Release Artifact Types
+
+### Docker Images
+
+*Tied to Code Release?* Yes
+
+Docker images are released for every named release of EdgeX Foundry. During development the community releases images to the `docker.staging` repository in [Nexus](http://nexus3.edgexfoundry.org). At the time of release we promote the last tested image from `docker.staging` to `docker.release`. In addition to that we will publish the docker image on [DockerHub](https://hub.docker.com/orgs/edgexfoundry).
+
+#### Nexus Retention Policy
+
+##### docker.snapshots
+Retention Policy: 90 days since last download
+
+Contains: Docker images that are not expected to be released. This will contain EdgeX that images are built with each merge and should be available to test things against a specific commit. In addition to this we also build images to optimize the builds in the CI infrastructure.
+
+Docker Tags Used: Git commit, Version-Git commit, Version (ci-build-images only), Latest (ci-build-images-only)
+
+##### docker.staging
+Retention Policy: 180 days since last download
+
+Contains: Docker images built for potential release and testing purposes.
+
+Docker Tags Used: Version (ie: v1.x), Release Branch (master, fuji, etc), Latest
+
+##### docker.release
+Retention Policy: No automatic removal. Requires TSC approval to remove images from this repository.
+
+Contains: Officially released docker images for EdgeX. 
+
+Docker Tags Used:•Version (ie: v1.x), Latest 
+
+[Nexus Cleanup Policies Reference](https://help.sonatype.com/repomanager3/cleanup-policies)
+
+### Docker Compose Files
+
+*Tied to Code Release?* Yes
+
+Docker compose files are released alongside the docker images for every release of EdgeX Foundry. During development the community maintains a compose file named `nightly-build`. This compose file is meant to be a beta version of EdgeX. At the time of release the community makes a named release compose file. (ie: `geneva`)
```

Comment:
I'm not sure what you mean by "this compose file is meant to be a beta version of EdgeX"? It's not really meant as a beta (which is concept that some projects/products to release an early preview of a release in order to solicit customer/user feedback**), it's meant to be the compose file used during the entire development cycle.

****** we should discuss at some point whether we should be producing actual release candidates and/or Beta releases

Reply:
Also, do we have rules on when & how released compose files can be modified?

Reply:
We've only ever updated released compose files tied to a release or a dot release. Otherwise it doesn't seem to get much action and there isn't any rules around it today.

I realize beta is a loaded term, but the crux of it is that those compose files are used by the testing frameworks to run tests every night. Because of that it means it is tracking the code that is in master branch of the services. Which makes it  more of a "beta"/"experimental"/"dev" release of EdgeX. I could swap it out for experimental is that is more clear.